ironfish/src/consensus/verifier.ts

```diff
@@ -179,11 +179,13 @@ export class Verifier {
       totalTransactionFees += transaction.fee()
     }
 
-    // minersFee should be (negative) miningReward + totalTransactionFees
-    const miningReward = this.chain.network.miningReward(block.header.sequence)
-
-    if (minersFeeTransaction.fee() !== -1n * (BigInt(miningReward) + totalTransactionFees)) {
-      return { valid: false, reason: VerificationResultReason.INVALID_MINERS_FEE }
+    // only verify mining reward if evm gas verified
+    if (options.verifyEvm) {
```

Comment:
Not fully understanding the reasoning for the conditional here

Reply:
Updated so that `verifyBlock` always executes the evm transactions, but it does so without affecting state.